internal/services/oracle/cloud_vm_cluster_resource.go

```diff
@@ -223,6 +226,14 @@ func (CloudVmClusterResource) Arguments() map[string]*pluginsdk.Schema {
 			ValidateFunc: validate.DataStoragePercentage,
 		},
 
+		"domain": {
+			Type:         schema.TypeString,
+			Optional:     true,
+			Computed:     true,
```

Comment:
Does the service generate a value for this if the user doesn't set it?

Reply:
yes, it's optional and control plane will return a generated value for it.

Reply:
@stephybun removed StringIsNotEmpty for optional arguments

Reply:
Is an empty string `""` a valid and accepted value for `domain` or `zone_id`?

Reply:
yes, it's accepted